### ISSUE TITLE
doesnt work in 2.2.4

### ISSUE THREAD

**Issue by gerlstar** [vz28bh/CakePHP-JqueryValidationHelper#1]
Hey I tried this with my version 2.2.4 but it looks like it's still validating serverside. CAn you please fix it?

**Comment by vz28bh** [comment]
Hi, thanks for trying it out.  Try updating one section of code after line 98 by updating to match this:

```
    $meta = $this->meta($model, $fieldName);
    $formatted_js = strtr($this->js, $map);
    $boolean_fix_map = array(
        '"true"' => 'true',
        '"false"' => 'false',
    );
    $options['data-validate'] = strtr(json_encode($meta, JSON_NUMERIC_CHECK), $boolean_fix_map);
    $response = $this->_addJs($map);
```

There is a problem with the way the metadata encodes booleans so I had to make an ugly hack to fix it.  I have a project in 2.2.4 that works correctly so hopefully that is the only issue.  Make a simple test first with a dialog that just has one field that should be numeric, then type an 'a' in the field and it should show an error message immediately.

I will make a commit on this shortly.

**Comment by aymanrb** [comment]
I am not sure if this worked for @gerlstar or not but it didn't work for me too.

After debugging for sometime on why it didn't work it turned out that JQueryValidation Plugin has dropped support for metadata plugin (that your helper relies on the most) - http://goo.gl/AY6omU

The class should use 'data-rule-' and 'data-msg-' instead :+1:
